shipyard-controller/controller/debugcontroller.go

```diff
@@ -0,0 +1,31 @@
+package controller
+
+import (
+	"github.com/gin-gonic/gin"
+	"github.com/keptn/keptn/shipyard-controller/handler"
+)
+
+type DebugController struct {
+	DebugHandler handler.IDebugHandler
+}
+
+func NewDebugController(debugHandler handler.IDebugHandler) Controller {
+	return &DebugController{DebugHandler: debugHandler}
+}
+
+func (controller DebugController) Inject(apiGroup *gin.RouterGroup) {
+	apiGroup.Static("/ui", "./debug-ui")
+
+	apiGroup.GET("/", func(c *gin.Context) {
+		c.Redirect(302, "/ui")
+	})
+
+	seq := apiGroup.Group("/debug")
```

Comment:
Since we are retrieving sequence info, I would expect this name
```suggestion
	seq := apiGroup.Group("/sequence")
```